TestAssets/TestProjects/MSBuildTestAppWithCorruptedLaunchSettings/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="test-packages" value="D:\Projects\cli\artifacts\testpackages" />
```

Comment:
this seems wrong. We already have a NuGet.Config in the testprojects folder for this, I am pretty sure. Plus, this should not be hard coded to D:\...